articles/openshift/concepts-restrict-egress-traffic.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Concepts - Controlling and restricting egress traffic
+description: How to control and restrict egress traffic from Azure Red Hat OpenShift 4 clusters
+author: sakthi-vetrivel
+ms.author: suvetriv
+ms.topic: concept
+ms.service: container-service
+ms.date: 07/23/2020
+#Customer intent: As a cluster administrator, I want to be able to restrict and control egress traffic from my cluster.
+---
+
+# Controlling and restricting egress traffic in Azure Red Hat OpenShift 4
+
+This article provides the necessary details that allow you to secure outbound traffic from your Azure Red Hat OpenShift 4 clusters.
+
+> [!IMPORTANT]
+> The specific network rules, endpoints, ports, and other information provided in this document is subject to change.
+
+It contains the cluster requirements for a base AKS deployment, and additional requirements for optional addons and features. [An example will be provided at the end on how to configure these requirements with Azure Firewall](#restrict-egress-traffic-using-azure-firewall). However, you can apply this information to any outbound restriction method or appliance.
```

Comment:
s/AKS/ARO/g

Remove the Link to Azure firewall since this isn't a well known documented case.

Reply:
I think this is a scenario (Azure Firewall) that we can / should test.  We have a customer who wants to do this with a ZScaler, so this might be the closest thing we can do to verify.